images.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Image struct {
+	Name string
+	Tags []Tag
+}
+
+type Tag struct {
+	Sha string
+	Tag string
+}
+
+var Images = []Image{
+	Image{
+		Name: "quay.io/coreos/hyperkube",
```

Comment:
Only adding hyperkube for the moment, I'll move everything we need once this PR is sorted